packages/test-bundles/build.ts

```diff
@@ -2,7 +2,7 @@ import react from "@vitejs/plugin-react-swc";
 import fs from "fs";
 import { join, parse } from "path";
 import { visualizer } from "rollup-plugin-visualizer";
-import { fileURLToPath } from "url";
+import { URL, fileURLToPath } from "url";
```

Comment:
Surprised this one isn't coming in from the DOM bit of `"lib": ["es2020", "DOM"]`, but maybe I'm misunderstanding that option!

Reply:
Same, i think this is just how the node types are, the global `URL` is actually a different, incompatible type from this one